pkg/cluster/spec/prometheus.go

```diff
@@ -253,6 +255,15 @@ func (i *MonitorInstance) InitConfig(
 		}
 	}
 
+	globalConfig := i.topo.(*Specification).ServerConfigs.Prometheus
```

Comment:
This type assert may panic when deploy dm cluster, suggest remove the global Prometheus config since we only support one prometheus.

Reply:
I had removed the global Prometheus config. But, will prometheus be deployed in cluster mode in the future to ensure high availability?

Reply:
Likely, but don't worry about it now

Reply:
In my last job, prometheus has made highly available deploy. haha